Documentation/network/servicemesh/gateway-api/splitting.rst

```diff
@@ -0,0 +1,143 @@
+This is very useful for canary testing or A/B scenarios.
+
+For this particular use case, we're going to use Gateway API to load-balance incoming traffic to different backends, starting with the same weights before testing with a 99/1 weight distribution.
+
+.. include:: ../echo-app.rst
+
+Deploy the Cilium Gateway
+=========================
+
+You'll find the example Gateway and HTTPRoute definition in ``splitting.yaml``.
+
+.. literalinclude:: ../../../../examples/kubernetes/gateway/splitting.yaml
+
+Note the even 50/50 split between the two Services. Deploy the Gateway and the HTTPRoute:
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/gateway/splitting.yaml
+
+The above example creates a Gateway named ``cilium-gw`` that listens on port 80.
+A single route is defined and includes two different ``backendRefs`` (``echo-1`` and ``echo-2``) and weights associated with them.
+
+.. code-block:: shell-session
+
+    $ kubectl get gateway cilium-gw
+    NAME        CLASS    ADDRESS          PROGRAMMED   AGE
+    cilium-gw   cilium   172.18.255.200                8s
+
+.. Note::
+
+    Some providers e.g. EKS use a fully-qualified domain name rather than an IP address.
+
+Even traffic split
+==================
+
+Now that the Gateway is ready, you can make HTTP requests to the services.
+
+.. code-block:: shell-session
+
+    $ GATEWAY=$(kubectl get gateway cilium-gw -o jsonpath='{.status.addresses[0].value}')
+    $ curl --fail -s http://$GATEWAY/echo
+
+    Hostname: echo-1-7d88f779b-m6r46
+
+    Pod Information:
+        node name:      kind-worker2
+        pod name:       echo-1-7d88f779b-m6r46
+        pod namespace:  default
+        pod IP: 10.0.2.15
+
+    Server values:
+        server_version=nginx: 1.12.2 - lua: 10010
+
+    Request Information:
+        client_address=10.0.2.252
+        method=GET
+        real path=/echo
+        query=
+        request_version=1.1
+        request_scheme=http
+        request_uri=http://172.18.255.200:8080/echo
+
+    Request Headers:
+        accept=*/*  
+        host=172.18.255.200  
+        user-agent=curl/7.81.0  
+        x-forwarded-proto=http  
+        x-request-id=ee152a07-2be2-4539-b74d-ebcebf912907  
+
+    Request Body:
+        -no body in request-
+
+Notice that, in the reply, you get the name of the pod that received the query. For example:
+
+.. code-block:: shell-session
+
+    Hostname: echo-2-5bfb6668b4-2rl4t
+
+Repeat the command several times.
+You should see the reply being balanced evenly across both pods/nodes.
+Let's double check that traffic is evenly split across multiple Pods by running a loop and counting the requests:
+
+.. code-block:: shell-session
+
+    while true; do curl -s -k "http://$GATEWAY/echo" >> curlresponses.txt ;done
+
+Stop with ``Ctrl+C``.
+Verify that the responses have been (more or less) evenly spread.
```

Comment:
Use present tense, avoid unnecessary parentheticals, clarity for easier localization
```suggestion
Verify that the responses are more or less evenly distributed.
```